frontend/public/components/resource-dropdown.tsx

```diff
@@ -106,7 +113,7 @@ const ResourceListDropdown_: React.SFC<ResourceListDropdownProps> = (props) => {
       return false;
     }
 
-    return fuzzy(_.toLower(text), _.toLower(model.kind));
+    return fuzzy(_.toLower(text), model.labelKey ? t(model.labelKey) : _.toLower(model.kind));
```

Comment:
You'll want to `toLower` the translated value too. Otherwise it will *never* match since we're calling `_.toLower` on `text`. It might be good to use `toLocaleLowerCase`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleLowerCase

Reply:
Good call; thanks for the tip.